src/utils/time.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Returns a string based on a give timestamp that represents relative time e.g.:
+ * - just now
+ * - 10 seconds ago
+ * - 2 hours ago
+ * @param {string} timestamp Timestamp in seconds
+ * @return { string } String representing relative time
+ */
+export const getRelativeTime = (timestamp: number): string => {
+  // Get the current time in seconds
+  const now = Math.floor(Date.now() / 1000)
+
+  // Calculate the time difference in seconds
+  const secondsAgo = now - timestamp
+
+  // Define an array of time units, each with a name and number of seconds
+  const timeUnits = [
+    { unit: "year", seconds: 31536000 },
+    { unit: "month", seconds: 2592000 },
+    { unit: "day", seconds: 86400 },
+    { unit: "hour", seconds: 3600 },
+    { unit: "minute", seconds: 60 },
+    { unit: "second", seconds: 1 },
+  ]
```

Comment:
Let's move this array out of the `getRelativeTime` function scope.

Reply:
db774a9f41d85f7fef0ad9a3e4c62b55b5caff9e